modules/python/python-grammar.ym

```diff
@@ -94,6 +94,7 @@ python_dd_option
             python_dd_set_imports(last_driver, $3);
           }
         | KW_OPTIONS '(' python_dd_custom_options ')'
+        | threaded_dest_driver_option
```

Comment:
Here is my analysis:
The list of log threaded destinations:
afamqp, afmongodb, afsmtp, afstomp, http, java, python, redis, riemann
All these contain threaded_dest_driver_option in their grammar file. Except python which will be fixed by this patch.